datamol/conformers/_conformers.py

```diff
@@ -125,20 +133,28 @@ def generate(
     params = getattr(AllChem, method)()
     params.randomSeed = random_seed
     params.enforceChirality = True
+    if custom_embed_params is not None:
+        for k, v in custom_embed_params.items():
+            setattr(params, k, v)
+
     confs = AllChem.EmbedMultipleConfs(mol, numConfs=n_confs, params=params)
 
     # Sometime embedding fails. Here we try again by disabling `enforceChirality`.
     if len(confs) == 0:
         if verbose:
             logger.warning(
-                f"Conformers embedding failed for {dm.to_smiles(mol)}. Trying without enforcing chirality."
+                f"Conformers embedding failed for {dm.to_smiles(mol)}. Trying without enforcing chirality and with random coordinates (if set)."
             )
         params = getattr(AllChem, method)()
         params.randomSeed = random_seed
```

Comment:
I feel like we should re-inject `custom_embed_params` here. At least I would expect this. What do you think?

If we do, I guess it should be before we write the fallback parameters.

Reply:
Right, I didn't see that you reset the params. Can we just not reset the params in that case and only change the relevant ones?

Reply:
I've tested this, params is not locked in any way so it should not be an issue.